python/pysixtracklib/beam_elements.py

```diff
@@ -288,9 +288,15 @@ def get(self, objid):
         return self.cbuffer.get_object(objid)
 
     @classmethod
-    def from_mad(cls, seq):
+    def from_mad(cls, seq, drift_exact=True):
```

Comment:
I would not default to exact=True, because it is much slower and not needed in most of the use cases.

Reply:
changed :+1: